src/main/kotlin/no/nav/klage/oppgave/api/view/OversendtKlage.kt

```diff
@@ -53,6 +54,57 @@ data class OversendtKlage(
     )
 }
 
+class HjemmelFraFoersteInstans private constructor(
+    val kapittel: Int?,
+    val paragraf: Int?,
+    val lov: Lov,
+) {
+    constructor(lov: Lov) : this(null, null, lov)
+    constructor(lov: Lov, kapittel: Int) : this(kapittel, null, lov)
+    constructor(lov: Lov, kapittel: Int, paragraf: Int) : this(kapittel, paragraf, lov)
+
+    override fun toString(): String {
+        if (kapittel != null && paragraf != null) {
+            return "$lov $kapittel-$paragraf"
+        } else if (kapittel != null) {
+            return "$lov $kapittel"
+        } else {
+            return "$lov"
+        }
+    }
+
+    companion object {
+
+        @Suppress("JAVA_CLASS_ON_COMPANION")
+        private val logger = getLogger(javaClass.enclosingClass)
+
```

Comment:
Hva brukes denne til? Får vi hjemmel som string fra noe sted nå? Var ikke det bare fra Oppgave?

Reply:
Du har lagt kommentaren på loggeren, men jeg antar at du mener fromString-metoden?
Jeg har ikke endret hvordan hjemlene på Mottak lagres, så der er det fremdeles en String, ikke en List<HjemmelFraFoersteInstans> eller for den saks skyld en List<String>.. Så når vi leser opp fra db, så må vi parse den Stringen..

Reply:
Skal vi endre lagringen kanskje? Jeg ser at MottakDokument har dukket opp nå, så kanskje MottakHjemmel hadde gjort seg også da. Å lagre en liste som en String oppleves som et hack..

Reply:
Jeg gjør det jeg!

Reply:
Fikset